src/screens/Podcast/Podcast.tsx

```diff
@@ -82,7 +82,7 @@ const Podcast: React.FC = () => {
   }
 
   const handleNavigateToHome = () => {
```

Comment:
If you navigate back, let's name it in accordance, please.
```suggestion
  const handleNavigateBack = () => {
```